app/Ship/core/Generator/Stubs/transporter.stub

```diff
@@ -16,8 +16,8 @@ class {{class-name}} extends Transporter
             // enter all properties here
 
             // allow for undefined properties
-            // 'additionalProperties' => true,
         ],
+        'additionalProperties' => true,
```

Comment:
I think 'additionalProperties' default value should be 'false' to preserve previous behaviour, as it was previously commented (disbaled) by default.